tests/unit/log_printer_test.py

```diff
@@ -40,11 +41,30 @@ def reader(*args, **kwargs):
 
         self.assertIn(glyph, output)
 
+    def test_notail(self):
+        def reader(*args, **kwargs):
+            for count in xrange(3):
+                yield "keep going\n"
+                time.sleep(1)
```

Comment:
I always try to avoid any `sleep()` in tests. It slows things down unnecessarily.

Is this actually required for the tests to pass?